### ISSUE TITLE
goroutine stack exceeds 1000000000-byte limit

### ISSUE THREAD

**Issue by fahmifan** [uudashr/gocognit#3]
I run gocognit `v1.0.2`  on `gocognit` repository branch master and got `goroutine stack exceeds 1000000000-byte limit` 
```
go run cmd/gocognit/main.go -over 15 .
runtime: goroutine stack exceeds 1000000000-byte limit
runtime: sp=0xc0201e0388 stack=[0xc0201e0000, 0xc0401e0000]
fatal error: stack overflow

runtime stack:
runtime.throw(0x113aa8e, 0xe)
```

But, run it on small codebase like [this](https://github.com/fahmifan/lrucache) doesn't get the error

`go version go1.16.5 darwin/amd64`

**Comment by fahmifan** [comment]
Unfortunately, the issue still  happen after update it to v1.0.4

**Comment by uudashr** [comment]
Fixed on v1.0.5. Please confirm if it works on your side.

**Comment by fahmifan** [comment]
Nice, the issue is fixed on `v1.0.5` 👍
